common/path.go

```diff
@@ -64,12 +64,11 @@ func AbsolutePath(Datadir string, filename string) string {
 }
 
 func HomeDir() (home string) {
-	if usr, err := user.Current(); err == nil {
+	home = os.Getenv("HOME")
+	if usr, err := user.Current(); err == nil && home == "" {
```

Comment:
This `if` is a bit strange here btw, because you retrieve the current user, even if home is already set, and discard the user afterwards. Probably a cleaner code would be to short circuit execution immediately if HOME is set:

```
if home := os.Getenv("HOME"); home != "" {
	return home
}
if usr, err := user.Current(); err == nil {
	return usr.HomeDir
}
return ""
```

I would also drop the `home` from the return "type", so that the header simply becomes 

```
func HomeDir() string
```

The older parts of the code use occasionally these named returns, but they usually over complicate code and rarely provide any additional advantage, so it's usually cleaner to not have them. Omitting them also avoids issues where you accidentally return something else than you'd expect/want due to some variable shadowing.